workshops/util.py

```diff
@@ -46,22 +43,9 @@ def upload_person_task_csv(uploaded_file, encoding=None):
 
     Also return a list of fields from Person.PERSON_UPLOAD_FIELDS for which
     no data was given.
-
-    :param string encoding: encoding used to encode incoming file. Defaults to
-                            ``django.conf.settings.DEFAULT_CHARSET``
     """
     persons_tasks = []
 
-    if not encoding:
-        encoding = settings.DEFAULT_CHARSET
-
-    # we provide uploaded_file as StringIO in our tests (test_util.py)
-    if not issubclass(StringIO, uploaded_file.__class__):
-        # Django provides uploaded files as byte file objects.  We need to wrap
-        # `uploaded_file` and provide it as a text file with specific encoding.
-        # This way we force users to upload UTF-8 encoded files.
-        uploaded_file = TextIOWrapper(uploaded_file.file, encoding=encoding)
-
     reader = csv.DictReader(uploaded_file)
```

Comment:
On Thu, Feb 05, 2015 at 02:08:31PM -0800, Piotr Banaszkiewicz wrote:
> Anyway, that's what code review is, right? We take a critical look
> at each others code.

Yeah, I was just saying that “Yeah, you're right, sorry. I didn't
activate…” could have been “Oops, I didn't activate…”.  With all the
rest there, I just thought I'd make sure you weren't actually thinking
that the push-back bothered me.  Anyhow, it sounds like we were on the
same page from the start ;).